src/bun.js/api/bun/subprocess.zig

```diff
@@ -717,7 +715,8 @@ pub const Subprocess = struct {
         }
     }
 
-    pub fn doSend(this: *Subprocess, global: *JSC.JSGlobalObject, callFrame: *JSC.CallFrame) JSValue {
+    pub fn doSend(this: *Subprocess, global: *JSC.JSGlobalObject, callFrame: *JSC.CallFrame) callconv(.C) JSValue {
+        Output.scoped(.IPC, false)("Subprocess#doSend\n", .{});
```

Comment:
Please don't declare the IPC scope multiple times. It's going to cause weird bugs because it relies on comptime memoization to dedupe itself

Also, `#doSend\n` shouldn't be necessary, it should always add a newline at the end